extensions/wikia/AnalyticsEngine/js/universal_analytics.js

```diff
@@ -350,7 +350,7 @@
 		['set', 'dimension17', window.wgWikiVertical],                          // Vertical
 		['set', 'dimension18', window.wgWikiCategories.join(',')],              // Categories
 		['set', 'dimension19', window.wgArticleType],                           // ArticleType
-		['set', 'dimension20', window.wgABPerformanceTest || 'not set'],        // Performance A/B testing
+		['set', 'dimension20', 'not set'],        								// Performance A/B testing (Not used any more)
```

Comment:
I'd even remove it instead of setting it to `'not set'` but let me learn if it's even being used by somebody. Not a blocker, though.